setup/nginx-web/nginx-install-ubuntu.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+sudo apt install -y curl gnupg2 ca-certificates lsb-release ubuntu-keyring
+curl https://nginx.org/keys/nginx_signing.key | gpg --dearmor \
```

Comment:
Currently the script is not idempotent. (If i run it second time It fails) Nice to have the ability to run it multiple times without side effects